docs/api/apiv3/components/schemas/storage_model.yml

```diff
@@ -0,0 +1,59 @@
+# Schema: StorageModel
+---
+type: object
+required:
+  - id
+  - _type
+  - href
+  - storageType
+properties:
+  id:
+    type: integer
+    description: Storage id
+  _type:
+    type: string
+    enum:
+      - Storage
+  href:
+    type: string
+    format: uri
+    description: web uri of the storage instance
+  storageType:
+    type: string
+    enum:
+      - nextcloud
```

Comment:
We could also introduce a uri for this to highlight the fact that this is actually a constant. Then, the storageType would be moved to the `_links` section and could also receive a `title` attribute.

Reply:
Is there any benefit from moving constants to uris? why is a resource behind a uri more likely a constant, than an enumeration string?

Reply:
moved to _links as an urn.